README.md

```diff
@@ -1,27 +1,40 @@
-# DASK_GDF
+# Dask GPU Dataframes
 
-A partitioned gpu-backed dataframe, using dask.
+A partitioned gpu-backed dataframe, using Dask.
 
-## Setup
+## Setup from source
 
 Setup from source repo:
 
-1. Follow instructions in https://github.com/gpuopenanalytics/pygdf#setup to setup
-   a conda environment for pygdf.
-1. Activate pygdf environment: `source activate pygdf_dev`
-1. Clone dask_gdf repo: `git clone https://github.com/gpuopenanalytics/dask_gdf path/to/dask_gdf`
-1. `cd path/to/dask_gdf`
-1. Install additional dependency: `conda install dask distributed`
-
-Installing (Optional):
-1. Run
-   ```bash
-   cd path/to/dask_gdf
-   pip install .
-   ```
-
-Testing:
-1. Ensure pytest is available: `conda install pytest`
-1. `cd path/to/dask_gdf`
-1. Run all tests with: `pytest dask_gdf`
-1. Or, run individual tests with: `pytest dask_gdf/tests/test_file.py`
+1.  Install dependencies into a new conda environment
+
+        conda install -n dask-gdf \
+           -c numba -c conda-forge -c gpuopenanalytics/label/dev -c defaults \
+           pygdf dask distributed cudatoolkit
```

Comment:
Many users will already have CUDA installed from outside of conda so we should note installing `cudatoolkit` from conda isn't required in that case.